accounts/abi/type.go

```diff
@@ -57,16 +58,16 @@ var (
 	// Types can be in the format of:
 	//
 	// 	Input  = Type [ "[" [ Number ] "]" ] Name .
-	// 	Type   = [ "u" ] "int" [ Number ] .
+	// 	Type   = [ "u" ] "int" [ Number ] [ x ] [ Number ].
```

Comment:
Also this type "description" doesn't contain strings, neither fixed types in them, so perhaps we could rather split the `Type` variations onto three lines describing the ints, strings and fixes. (also bytes?)